ocean_lib/assets/__init__.py

```diff
@@ -1,4 +0,0 @@
-#
```

Comment:
Why was this deleted? I know that for pytest (and maybe other things) you need an __init__.py file.